Makefile

```diff
@@ -333,12 +339,9 @@ docker-push: ## Push the docker images
 	docker push $(MANIFEST_IMG):$(TAG)
 
 .PHONY: docker-push-all
-docker-push-all: $(addprefix docker-push-,$(ALL_ARCH))  ## Push all the architecture docker images
+docker-push-all: docker-push  ## Push all the architecture docker images
 	$(MAKE) docker-push-manifest-rancher-turtles
 
-docker-push-%:
-	$(MAKE) ARCH=$* docker-push
-
 .PHONY: docker-push-manifest-rancher-turtles
 docker-push-manifest-rancher-turtles: ## Push the multiarch manifest for the rancher turtles docker images
```

Comment:
I don't think we need ` docker-push-manifest-rancher-turtles` anymore, it will done by `buildx`

Reply:
Yes, good catch. I'm still cleaning up release workflow and checking what actions we can remove.